ParseLoginUI/src/main/java/com/parse/ParseQueryRecyclerViewAdapter.java

```diff
@@ -0,0 +1,611 @@
+package com.parse;
```

Comment:
Also, if we move it to a different package we cant use package-private methods:
```ParseObject.getClassName(String)```, ```Parse.isLocalDatastoreEnabled()```
Can I make them public?